cmd/svcat/class/create_cmd.go

```diff
@@ -57,25 +68,33 @@ func (c *CreateCmd) Validate(args []string) error {
 		return fmt.Errorf("new class name should be provided")
 	}
 
+	if c.Namespace != "" && c.Cluster {
```

Comment:
Let me know if you copied this validation from somewhere else because I think it's going to run into trouble once you switch to using `Namespaced` . `c.Namespace` will be populated with the _effective_ namespace, which could be the current namespace specified in the kubeconfig context.

The way the cluster flag works is that it takes precedence over the namespace flag. So if you specify `--cluster` the namespace is ignored, and it's not an error that we need to check for.